grid2op/l2rpn_utils/idf_2023.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2019-2020, RTE (https://www.rte-france.com)
+# See AUTHORS.txt
+# This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0.
+# If a copy of the Mozilla Public License, version 2.0 was not distributed with this file,
+# you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+# This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
+from grid2op.Action import PlayableAction
+from grid2op.Observation import CompleteObservation
+
+class ActionIDF2023(PlayableAction):
+    authorized_keys = {
+        "set_line_status",
+        "change_line_status",
+        "set_bus",
+        "change_bus",
+        "redispatch",
+        "set_storage",
+        "curtail",
+        "raise_alert",
+        }
+
+    attr_list_vect = [
+        "_set_line_status",
+        "_switch_line_status",
+        "_set_topo_vect",
+        "_change_bus_vect",
+        "_redispatch",
+        "_storage_power",
+        "_curtail",
+        "_raise_alert",
+        ]
+    attr_list_set = set(attr_list_vect)
+    pass
+    
+class ObservationIDF2023(CompleteObservation):
+    attr_list_vect = [
```

Comment:
TODO check there is all properties of the CompleteObs except the Alarm stuff !

Reply:
it seems correct